### PR TITLE
Chana/fix linting after main history rewrite

### PR DESCRIPTION
**Pull request by juanigaray**

### CODE REVIEW

File: src/components/Analisis.jsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import styles from "./Analisis.module.css";
 import { Link as ScrollLink } from "react-scroll";
 import PropTypes from "prop-types";
 
-export default function Analisis({ min, max, total, tipos, componentes }) {
+export default function Analisis({ min, max, total, componentes }) {
```
Comment (xaiki) [inline src/components/Analisis.jsx:5]: shouldn't tipos be removed from analisisData too ?

File: src/components/Analisis.jsx [hunk 2]
```diff
@@ -27,7 +27,6 @@ export default function Analisis({ min, max, total, tipos, componentes }) {
       </div>
 
       <div className={styles.analisisDatos}>
-
```
Comment (xaiki) [inline src/components/Analisis.jsx:30]: was this required by the linter ?

File: src/components/Analisis.jsx (hunk 1 above)
Comment (juanigaray) [reply src/components/Analisis.jsx:5]: Ah, yes. Probably gonna find a lot more like these with Typescript. I'll push a new commit for this.

File: src/components/Analisis.jsx (hunk 2 above)
Comment (juanigaray) [reply src/components/Analisis.jsx:30]: Yeah, it's the thing that's breaking in the latest PRs

**Pull request merged**